source/tests/system/nisync_driver_api_tests.cpp

```diff
@@ -462,6 +462,100 @@ class NiSyncDriverApiTest : public ::testing::Test {
+    request.set_time_seconds(timeSeconds);
+    request.set_time_nanoseconds(timeNanoseconds);
+    request.set_time_fractional_nanoseconds(timeFractionalNanoseconds);
+    request.mutable_vi()->set_id(driver_session_->id());
+    auto grpcStatus = GetStub()->CreateFutureTimeEvent(&clientContext, request, &response);
+    *viStatusOut = response.status();
+    return grpcStatus;
+  }
+ 
+  ::grpc::Status call_ClearFutureTimeEvents(
+     ViConstString terminalName,
+     ViStatus* viStatusOut)
+  {
+    ::grpc::ClientContext clientContext;
+    nisync::ClearFutureTimeEventsRequest request;
+    nisync::ClearFutureTimeEventsResponse response;
+    request.set_terminal_name(terminalName);
+    request.mutable_vi()->set_id(driver_session_->id());
+    auto grpcStatus = GetStub()->ClearFutureTimeEvents(&clientContext, request, &response);
+    *viStatusOut = response.status();
+    return grpcStatus;
+  }
+
+  ::grpc::Status call_CreateClock(
+     ViConstString terminalName,
+     ViUInt32 highTicks,
+     ViUInt32 lowTicks,
+     ViUInt32 startTimeSeconds,
+     ViUInt32 startTimeNanoseconds,
+     ViUInt16 startTimeFractionalNanoseconds,
+     ViUInt32 stopTimeSeconds,
+     ViUInt32 stopTimeNanoseconds,
+     ViUInt16 stopTimeFractionalNanoseconds,
+     ViStatus* viStatusOut)
+  {
+    ::grpc::ClientContext clientContext;
+    nisync::CreateClockRequest request;
+    nisync::CreateClockResponse response;
+    request.set_terminal_name(terminalName);
+    request.set_high_ticks(highTicks);
+    request.set_low_ticks(lowTicks);
+    request.set_start_time_seconds(startTimeSeconds);
+    request.set_start_time_nanoseconds(startTimeNanoseconds);
+    request.set_start_time_fractional_nanoseconds(startTimeFractionalNanoseconds);
+    request.set_stop_time_seconds(stopTimeSeconds);
+    request.set_stop_time_nanoseconds(stopTimeNanoseconds);
+    request.set_stop_time_fractional_nanoseconds(stopTimeFractionalNanoseconds);
+    request.mutable_vi()->set_id(driver_session_->id());
+    auto grpcStatus = GetStub()->CreateClock(&clientContext, request, &response);
+    *viStatusOut = response.status();
+    return grpcStatus;
+  }
+
+  ::grpc::Status call_ClearClock(
+     ViConstString terminalName,
+     ViStatus* viStatusOut)
+  {
+    ::grpc::ClientContext clientContext;
+    nisync::ClearClockRequest request;
+    nisync::ClearClockResponse response;
+    request.set_terminal_name(terminalName);
+    request.mutable_vi()->set_id(driver_session_->id());
+    auto grpcStatus = GetStub()->ClearClock(&clientContext, request, &response);
+    *viStatusOut = response.status();
+    return grpcStatus;
+  }
+
+  ViUInt32 GetCurrentBoardTimeSeconds()
+  {
+    ViStatus viStatus;
+    ViUInt32 timeSeconds = 0, timeNanoseconds;
+    ViUInt16 timeFractionalNanoseconds;
+    auto grpcStatus = call_GetTime(
+      &timeSeconds,
+      &timeNanoseconds, // ignored
+      &timeFractionalNanoseconds, // ignored
+      &viStatus);
```

Comment:
I don't know why this would fail, and other stuff is probably going to fail if this does, but should we be asserting on grpcStatus and viStatus here?

```
  EXPECT_TRUE(grpcStatus .ok());
  EXPECT_EQ(VI_SUCCESS, viStatus);
```